WooCommerce/WooCommerceTests/POS/ViewHelpers/PointOfSaleOrderDetailsViewHelperTests.swift

```diff
@@ -0,0 +1,471 @@
+import Foundation
+import Testing
+@testable import WooCommerce
+import Yosemite
+import WooFoundation
+
+struct PointOfSaleOrderDetailsViewHelperTests {
+    private let mockCurrencySettings: CurrencySettings
+    private let sut: PointOfSaleOrderDetailsViewHelper
+
+    init() {
+        mockCurrencySettings = CurrencySettings()
+        sut = PointOfSaleOrderDetailsViewHelper(currencySettings: mockCurrencySettings)
+    }
+
+    // MARK: - Products Subtotal Tests
+
+    @Test func productsSubtotal_emptyLineItems_returnZero() async throws {
```

Comment:
I'd add the "formatted" bit to the test name: 

```suggestion
    @Test func productsSubtotal_when_empty_line_items_then_returns_formatted_zero_subtotal() async throws {
```